pkg/cli/cmds/server.go

```diff
@@ -208,20 +209,25 @@ var ServerFlags = []cli.Flag{
 	ClusterDomain,
 	cli.StringFlag{
 		Name:        "flannel-backend",
-		Usage:       "(networking) backend<=option1=val1,option2=val2> where backend is one of 'none', 'vxlan', 'ipsec', 'host-gw', 'wireguard-native', or 'wireguard' (deprecated)",
+		Usage:       "(deprecated) use --flannel-opts",
 		Destination: &ServerConfig.FlannelBackend,
 		Value:       "vxlan",
 	},
 	cli.BoolFlag{
 		Name:        "flannel-ipv6-masq",
-		Usage:       "(networking) Enable IPv6 masquerading for pod",
+		Usage:       "(deprecated) use --flannel-opts",
 		Destination: &ServerConfig.FlannelIPv6Masq,
 	},
 	cli.BoolFlag{
 		Name:        "flannel-external-ip",
-		Usage:       "(networking) Use node external IP addresses for Flannel traffic",
+		Usage:       "(deprecated) use --flannel-opts",
 		Destination: &ServerConfig.FlannelExternalIP,
 	},
+	cli.StringSliceFlag{
+		Name:  "flannel-opts",
+		Usage: "(networking) Flannel options: backend:<option>, ipv6-masq, external-ip. Backend is one of 'none', 'vxlan', 'ipsec', 'host-gw' or 'wireguard-native'",
```

Comment:
We're going to drop ipsec as well, so lets just list the still-supported backends.
```suggestion
		Usage: "(networking) Flannel options: backend=string, ipv6-masq=bool, external-ip=bool. backend is one of 'none', 'vxlan', 'host-gw' or 'wireguard-native'",
```